components/lib/Head/Head.tsx

```diff
@@ -16,16 +17,16 @@ export const Head: FC<IHeadProps> = (props) => {
   return (
     <NextHead>
       {/* Primary Meta Tags */}
-      <title>{title}</title>
-      <meta name='title' content={title} />
+      <title>{replaceWithSpace(title)}</title>
```

Comment:
create a local variable const cleanTitle = replaceWithSpace(title) instead of repeating it everywhere